src/jit/gentree.cpp

```diff
@@ -6308,6 +6308,10 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
```

Comment:
Feel free to postpone replacing structDesc with a ptr if you don't want to club with this change.